tests/kafkatest/services/connect.py

```diff
@@ -279,12 +282,34 @@ def append_to_environment_variable(self, envvar, value):
             env_opts = "\"%s %s\"" % (env_opts.strip('\"'), value)
         self.environment[envvar] = env_opts
 
+    def append_filestream_connectors_to_classpath(self):
```

Comment:
Append and return. Easier to write than finding a name for it. Let me give it a try :)

Reply:
In a sense it is append, because we are re-using the existing classpath. Multiple export commands will be cumulative. We don't need to have one such command as long as any export call keeps the existing classpath.

Reply:
Given the above and after taking a second look I think I'll keep the current name to avoid any implementation details on the name. Indeed by looking at the code each individual command is an append to the classpath.